### ISSUE TITLE
False positive: `click.figma.com` in `category-ads-all.txt`

### ISSUE THREAD

**Issue by nima1024m** [Chocolate4U/Iran-v2ray-rules#73]
### Describe the problem

`click.figma.com` is currently listed in **category-ads-all.txt**, but this sub-domain is **not** used for tracking or advertising. It is the redirect domain that Figma uses in invitation emails (e.g., “Join my Figma file”). Blocking it breaks shared-link invitations and effectively disables collaboration features for anyone behind the Iran-v2ray-rules list.

### Evidence

* When you invite someone to a Figma project, the email link starts with `https://click.figma.com/...`.
* Removing the block restores normal collaboration behavior.
* The domain is **not** flagged in the [ v2fly's domain-list-community `category-ads-all` list](https://github.com/v2fly/domain-list-community/blob/master/data/category-ads-all) and Loyalsoldier's list, suggesting consensus that it is safe.

### Suggested fix

Please remove this line from **category-ads-all.txt**:

```
domain:click.figma.com
```

### Impact

* ✅ Users can again accept collaboration invitations.
* ❌ No new ad or tracking vectors are introduced (the domain is purely functional).

Thanks for maintaining this list!

**Comment by patterniha** [comment]
به من هم پیام داد راست میگه بلاک کردن click.figma.com  مشکلاتی ایجاد میکنه


این دامنه تو لیست hagezi هست 

پس باید لیست hagezi از لیست دامنه های تبلیغات حذف شه

///

@Chocolate4U 

بهتر نیست برای لیست تبلیغات همون لیست loyalsoldier  رو بزاری؟

**لیست loyalsoldier شامل لیست PersianBlocker هم هست و دامنه های تبلیغات فارسی رو هم پشتیبانی میکنه**
ضمن اینکه میزان خطاش خیلی کمه 

به غیر hegezi که خطا زیاد داره لیست GoodbyeAds هم خیلی وقته اپدیت نشده

///

خلاصه لیستایی که استفاده کردی جالب نیست و بهتره با لیست 
loyalsoldier
جایگزاری شه

**Comment by nima1024m** [comment]
سلام
ممنون از شما

من تو این مدت هم این لیست و هم لیست loyalsoldier رو خیلی زیاد تست کردم و واقعیت اینه که لیست loyalsoldier تو مسدود کردن تبلیغات خیلی بهتر عمل میکنه و خطای کمتری داره و حتی تو وب فارسی هم به خوبی عمل میکنه، من خودم به شخصه از لیست chocolate فقط برای آی پی های ایرانی استفاده میکنم و برای تبلیغات از همون loyalsoldier و اینطوری بهترین نتیجه رو تونستم بگیرم.

**Comment by Chocolate4U** [comment]
سلام
@nima1024m 
این دامنه رو Hagezi طی یک Issue یکی از کاربران به لیستش اضافه کرده. در کل هم دامنه Tracking هست ولی خب به دلیل مشکلات عملکردی که ایجاد میکنه، تو بقیه لیست ها نیست. لطفا اینجا [dns-blocklists/issues](https://github.com/hagezi/dns-blocklists/issues) مطرح کنید مشکل رو، معمولا خیلی زود برطرف میکنه.
@patterniha 
لیست Loyalsoldier خیلی سنگینه و اتفاقا خطای بیشتری داره. الان لیستش حدود 129 هزار دامنه داره که فقط 18 هزارتاش برای چین هست (لیست [easylistchina](https://github.com/easylist/easylistchina)) در حالی که لیست من حدود 65 هزارتا دامنه داره و خیلی سبکتره، با تقریبا همون عملکرد. لیست Persianblocker هم تا جایی که میدونم نداره.
لیست Hagezi هم جزو معتبرترین لیست های Adblock بر پایه DNS هست و توی اکثر DNS ها مثل ControlD، NextDNS و RethinkDNS داره استفاده میشه.

**Comment by patterniha** [comment]
ممنون حق با شماست
@nima1024m 
پس یه issue برای hagezi باید بزنی
